terraform/rack/aws/variables.tf

```diff
@@ -34,6 +34,10 @@ variable "oidc_sub" {
   type = string
 }
 
+variable "proxy_protocol" {
+  default = false
```

Comment:
@heronrs should this be true?  The docs suggest that the default is `true`.  Or should the docs change?